plugins/inputs/tail/tail_test.go

```diff
@@ -92,14 +96,13 @@ func TestTailDosLineendings(t *testing.T) {
 }
 
 func TestGrokParseLogFilesWithMultiline(t *testing.T) {
-	thisdir := getCurrentDir()
 	//we make sure the timeout won't kick in
 	duration, _ := time.ParseDuration("100s")
 
 	tt := NewTail()
 	tt.Log = testutil.Logger{}
 	tt.FromBeginning = true
-	tt.Files = []string{thisdir + "testdata/test_multiline.log"}
+	tt.Files = []string{filepath.Join(testdataDir, "test_multiline.log")}
```

Comment:
Again you are changing _current dir_ to _test data dir_. I guess also this instance is ok!?

Reply:
Exactly. I'm changing `current dir" + testdata dir` to `testdataDir`. Which is exactly the same :)

Reply:
Perfect. Resolved.